shell/platform/android/android_external_texture_gl.cc

```diff
@@ -64,7 +64,9 @@ void AndroidExternalTextureGL::Paint(SkCanvas& canvas,
       transformAroundCenter.postTranslate(0.5, 0.5);
       canvas.concat(transformAroundCenter);
     }
-    canvas.drawImage(image, 0, 0);
+    SkPaint paint;
+    paint.setFilterQuality(kLow_SkFilterQuality);
```

Comment:
This fix is making an assumption about which filter quality is desired. Should there be a way for the app to control the filter quality through the framework?

Reply:
Yes, providing a configurable option would be better. One should be able to add that option to engine `TextureLayer` and framework `Texture` widget similar to how to we added `clipBehavior` to engine `ClipPathLayer` and framework `ClipPath` widget.

If there's no such configurable option, I felt that the bilinear `kLow_SkFilterQuality` should be preferred over the nearest neighbor `kNone_SkFilterQuality`, unless the matrix has only translations (no perspective transformation and the scale is 1). Also CC @chinmaygarde for more thoughts.

Reply:
I agree that the default for the parameter should be FilterQuality.low.

Another point of reference for transferring enum data from the framework to the engine can be seen when the external-facing Paint object in lib/ui/painting.dart takes a FilterQuality parameter and converts it to an index which is then reconstructed into a native Skia filter value in lib/ui/painting/paint.cc

Reply:
ok, i will add it  as image's filter in this PR:https://github.com/flutter/flutter/pull/23329

Reply:
@flar @liyuqian Do i need to post another PR that include the framework change?

Reply:
Yes, I see you've already posted https://github.com/flutter/flutter/pull/58958

Reply:
@flar  are you still requiring some changes regarding the newest patch?